packages/spaces/src/lib/Spaces.stories.tsx

```diff
@@ -0,0 +1,19 @@
+// Each exported component in the package should have its own stories file
+
+import type { Meta, StoryObj } from '@storybook/react';
+import { Spaces, SpacesProps } from './Spaces';
+
+const meta: Meta<typeof Spaces> = {
+  title: 'Components/Spaces/Spaces',
+  component: Spaces,
+  tags: ['autodocs'],
+};
```

Comment:
looks like you're missing the QueryClientProvider decorator

Reply:
Added a story.